components/safe_browsing/android/java/src/org/chromium/components/safe_browsing/BraveSafeBrowsingApiHandler.java

```diff
@@ -142,12 +148,17 @@ public void startUriLookup(final long callbackId, String uri, int[] threatsOfInt
                                         == SafetyNetStatusCodes.SAFE_BROWSING_API_NOT_INITIALIZED) {
                             initSafeBrowsing();
                             startUriLookup(callbackId, uri, threatsOfInterest);
+                        } else {
+                            mObserver.onUrlCheckDone(callbackId, SafeBrowsingResult.TIMEOUT, "{}",
+                                    DEFAULT_CHECK_DELTA);
```

Comment:
Is `DEFAULT_CHECK_DELTA` in seconds?

Reply:
//   |check_delta| is the number of microseconds it took to look up the URL
//                 reputation from GmsCore.